tests/plugins/test_add_filesystem.py

```diff
@@ -36,6 +36,7 @@
     PreBuildPluginsRunner, PluginFailedException, BuildCanceledException)
 from atomic_reactor.plugins.pre_add_filesystem import AddFilesystemPlugin
 from atomic_reactor.util import ImageName, df_parser
+import atomic_reactor.util  # noqa
```

Comment:
`noqa` needed? If there's a particular warning to quell we ought to name it (e.g. "# noqa:E501")

Reply:
mmm maybe it isn't needed I've copy pasted it from somewhere